server/awsHelpers.js

```diff
@@ -60,13 +60,15 @@ const addSearchMessages = (messages) => {
   for (let i = 0; i < messages.length; i += 1) {
     generateRecommendation(messages[i]);
   }
+  console.log('search messages');
   const messagesArray = messages.map(message => generateRecommendation(message));
-  return Promise.all(messagesArray);
+  return Promise.all(messagesArray)
+    .then(allMessages => allMessages.filter(message => message.coefficients.priceCoefficient !== null));
 };
 
 module.exports.fetchSearchMessages = () => {
   const searches = [];
-  for (let i = 0; i < 300; i += 1) {
+  for (let i = 0; i < 1; i += 1) {
```

Comment:
Is it intentional to have this loop if this is only going to run once?

Reply:
Yes, the idea is that it will eventually loop through all 300. I have to implement other features before implementing that.